SolidStateGroup/bullet-train-api
# Add user name to confirmation dialog when removing users

Thanks for submitting a PR! Please check the boxes below:


## Changes

- Add user name to confirmation dialog when removing users

## How did you test this code?

- Click 'Manage' at the top of the application.
- Click 'Members' tab.
- Click remove button for any user.
- Check if the full name is displayed in the dialog.
